src/fides/lib/oauth/api/deps.py

```diff
@@ -0,0 +1,47 @@
+from typing import Generator
+
+from fastapi import Depends, Security
+from fastapi.security import SecurityScopes
+from sqlalchemy.orm import Session
+
+from fides.ctl.core.config import get_config
+from fides.lib.db.session import get_db_session
+from fides.lib.models.client import ClientDetail
+from fides.lib.oauth.api.urn_registry import TOKEN, V1_URL_PREFIX
+from fides.lib.oauth.oauth_util import verify_oauth_client as verify
+from fides.lib.oauth.schemas.oauth import OAuth2ClientCredentialsBearer
+
+
+def get_db() -> Generator[Session, None, None]:
+    """Return our database session.
+
+    This should be overridden by the installing package.
+    """
+    try:
+        SessionLocal = get_db_session(get_config())
+        db = SessionLocal()
+        yield db
+    finally:
+        db.close()
```

Comment:
There is a separate version of `get_db` in `src/fides/ops/api/api/deps.py` that used to be used to override this one. Now that they are using the same config we should be able to remove the one from ops.

Reply:
I'll take a look at consolidating these further, thanks!